File: content/code-examples/tested/java/driver-sync/timeseries/QuickStart/Tutorial.snippet.timefield-query.java
// Initialize date range
Date startTime = Date.from(Instant.parse("2021-12-18T15:50:00Z"));
Date endTime = Date.from(Instant.parse("2021-12-18T15:56:00Z"));

// Define the query filter
Document query = new Document("$and", Arrays.asList(
        new Document("date", new Document("$gte", startTime)),
        new Document("date", new Document("$lte", endTime))
));

FindIterable<Document> metaFieldResults = stocks.find(query)
        .projection(new Document("_id", 0));

for  (Document document : metaFieldResults) {
    System.out.println(document.toJson());
}

File: content/code-examples/tested/csharp/driver/TimeSeries/Tutorial.snippet.timefield-query.cs
// Initialize date range
var startTime = DateTime.Parse("2021-12-18T15:50:00Z");
var endTime = DateTime.Parse("2021-12-18T15:56:00Z");

// Define the query filter
var query = new BsonDocument("$and", new BsonArray
{
    new BsonDocument("date", new BsonDocument("$gte", startTime)),
    new BsonDocument("date", new BsonDocument("$lte", endTime))
});

var metaFieldResults = stocks.Find(query)
    .Project(Builders<Stocks>.Projection.Exclude("_id"))
    .ToEnumerable();

foreach (var document in metaFieldResults)
{
    Console.WriteLine(document.ToJson());
}

